### PR TITLE
feat: Allow custom configuration for keto k8s services

### PR DESCRIPTION
**Pull request by mickael-guesnon-3r**
Allow enabling/disabling the deployment of kubernetes services for the keto chart.
Allow custom naming of the keto services so it can be used with tools such as istio which require specific naming convention (protocol based) of the service ports in kubernetes.

## Checklist

- [X] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [X] I have read the [security policy](../security/policy).
- [X] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [X] I have added tests that prove my fix is effective or that my feature
      works.
- [X] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

File: docs/helm/charts/index.yaml [hunk 1]
```diff
@@ -3,9 +3,8 @@ entries:
   example-idp:
```
Comment (Demonsthere) [inline docs/helm/charts/index.yaml:3]: This file is autogenerated, please do not modify this

File: helm/charts/keto/Chart.yaml [hunk 2]
```diff
@@ -20,7 +20,7 @@ maintainers:
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.19.2
+version: 0.19.3
```
Comment (Demonsthere) [inline helm/charts/keto/Chart.yaml:23]: This is auto-modified on release, please do not modify this

File: helm/charts/keto/templates/service-read.yaml [hunk 3]
```diff
@@ -1,3 +1,4 @@
+{{- if.Values.service.enabled }}
```
Comment (Demonsthere) [inline helm/charts/keto/templates/service-read.yaml:1]:
```suggestion
{{- if.Values.service.read.enabled }}
```

**Review by Demonsthere — CHANGES_REQUESTED** [review]
Hello there!
Thanks for the PR, I left a few housekeeping comments ;)

**Comment by mickael-guesnon-3r** [comment]
@Demonsthere 
Thanks for the quick reply!
I applied changes as recommend and reverted the changes to the generated files.

Should I keep the tgz file in there?

**Comment by Demonsthere** [comment]
No, please delete the tgz, those are also regenerated on release :)

**Comment by mickael-guesnon-3r** [comment]
Done. Waiting for CI to turn green then requesting re-review.

**Review by Demonsthere — APPROVED** [review]
LGTM :)

**Pull request merged**